### PR TITLE
Flaskoman

### PR DESCRIPTION
**Pull request by tabreturn**
# New Port for Flaskoman

## Game Information
- **Title**: Flaskoman
- **URL**: https://store.steampowered.com/app/1645940/Flaskoman

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [ ] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/flaskoman/Flaskoman.sh [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+GAMEDIR=/$directory/ports/flaskoman
+
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs"
+export GMLOADER_DEPTH_DISABLE=1
+export GMLOADER_SAVEDIR="$GAMEDIR/gamedata/"
+export GMLOADER_PLATFORM="os_linux"
+
+# We log the execution of the script into log.txt
+exec > >(tee "$GAMEDIR/log.txt") 2>&1
```
Comment (JanTrueno) [inline ports/flaskoman/Flaskoman.sh:32]: This is the old loggin method, use this instead:
> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1

File: ports/flaskoman/Flaskoman.sh [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+cd $GAMEDIR
+
+if [ -f "${controlfolder}/libgl_${CFWNAME}.txt" ]; then
```
Comment (JanTrueno) [inline ports/flaskoman/Flaskoman.sh:36]: Gmloader is GLES, so no need to GL4ES.

**Review by JanTrueno — CHANGES_REQUESTED** [review]

File: ports/flaskoman/Flaskoman.sh (hunk 1 above)
Comment (tabreturn) [reply ports/flaskoman/Flaskoman.sh]: Amened. Thx

File: ports/flaskoman/Flaskoman.sh (hunk 2 above)
Comment (tabreturn) [reply ports/flaskoman/Flaskoman.sh]: Removed. Thx

**Review by JeodC — DISMISSED** [review]

**Comment by JeodC** [comment]
FYI I have an imagemaker tool that can generate mixv1 cover images for ports, if you want to include one.

**Review by Cebion — CHANGES_REQUESTED** [review]

**Review by JanTrueno — APPROVED** [review]

**Review by kloptops — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Pull request merged**